### PR TITLE
Inline shell commands

### PR DESCRIPTION
**Pull request by zeenix**
Mainly this PR is for inlining the various shell commands but some other minor fixes along too.

### CODE REVIEW

**Review by rpannek — APPROVED** [review]
Looks good to me :)

File: docs/chapters/baseplatform/using-the-repo-tool.rst [hunk 1]
```diff
@@ -8,13 +8,15 @@ The ``repo`` tool is used to obtain the sources of a PELUX release. ``repo`` was
 Obtaining the repo tool
 -----------------------
 
-If you already have a directory in your ``$PATH`` for placing user-specific runnables, you should use that location, if not perform the following steps:
+If you already have a directory in your ``$PATH`` for placing user-specific runnables, you should use that location; otherwise perform the following steps to add ~/bin (or any directory of your choice) to your ``$PATH``:
```
Comment (jonte) [inline docs/chapters/baseplatform/using-the-repo-tool.rst]: ``~/bin`` ?

Comment (jonte) [reply docs/chapters/baseplatform/using-the-repo-tool.rst]: Ah, shit. GH has markdown syntax support.. backticks for ~/bin ?

Comment (zeenix) [reply docs/chapters/baseplatform/using-the-repo-tool.rst]: that's the one being used in the example snippets

Comment (zeenix) [reply docs/chapters/baseplatform/using-the-repo-tool.rst]: ah ok, sure.

File: docs/chapters/baseplatform/obtaining-a-PELUX-release.rst [hunk 2]
```diff
@@ -14,22 +14,27 @@ Variables:
 
 Create a directory for the PELUX build. Instruct repo tool to fetch a manifest using the command `repo init`. In this context, branch denotes what branch of git repo `pelux-manifests` to use. Then make repo tool fetch all sources using the command `repo sync`.
 
-.. literalinclude:: snippets/repo-init.sh
-    :language: bash
+.. code-block:: bash
+
+  mkdir pelux
```
Comment (jonte) [inline docs/chapters/baseplatform/obtaining-a-PELUX-release.rst]: Indentation is not consistent with the other documents (please use 4 spaces)

Comment (zeenix) [reply docs/chapters/baseplatform/obtaining-a-PELUX-release.rst]: Sure.

**Review by jonte — APPROVED** [review]
Thanks for the updates!

**Review by breakreturn — APPROVED** [review]

**Pull request merged**